keyboards/keychron/k3/rgb/v2/optical_iso/keymaps/default/keymap.c

```diff
@@ -0,0 +1,172 @@
+    +------------------------------------------------------------------------------+
+*/
+  [MAC_BASE] = LAYOUT_75_iso(
+  /*  0          1          2          3        4        5        6         7        8        9          10          11          12         13         14         15        */
+      KC_ESC,    KC_BRID,   KC_BRIU,   KC_MSSN, KC_LAUN, RGB_VAD, RGB_VAI,  KC_MPRV, KC_MPLY, KC_MNXT,   KC_MUTE,    KC_VOLD,    KC_VOLU,   KC_MSNP,   KC_DEL,    RGB_TOG   ,
+      KC_GRV,    KC_1,      KC_2,      KC_3,    KC_4,    KC_5,    KC_6,     KC_7,    KC_8,    KC_9,      KC_0,       KC_MINS,    KC_EQL,    KC_BSPC,              KC_PGUP   ,
+      KC_TAB,    KC_Q,      KC_W,      KC_E,    KC_R,    KC_T,    KC_Y,     KC_U,    KC_I,    KC_O,      KC_P,       KC_LBRC,    KC_RBRC,                         KC_PGDN   ,
+      KC_CAPS,   KC_A,      KC_S,      KC_D,    KC_F,    KC_G,    KC_H,     KC_J,    KC_K,    KC_L,      KC_SCLN,    KC_QUOT,    KC_NUHS,   KC_ENT,               KC_HOME   ,
+      KC_LSFT,   KC_NUBS,   KC_Z,      KC_X,    KC_C,    KC_V,    KC_B,     KC_N,    KC_M,    KC_COMM,   KC_DOT,     KC_SLSH,               KC_RSFT,   KC_UP,     KC_END    ,
+      KC_LCTL,   KC_LGUI,   KC_LALT,                              KC_SPC,                                KC_RALT,   MO(MAC_FN), KC_RCTL,    KC_LEFT,   KC_DOWN,   KC_RGHT
+  ),
+
+/*  Mac Fn overlay
+    +------------------------------------------------------------------------------+
+    |    | F1 | F2 | F3 | F4 | F5 | F6 | F7 | F8 | F9 | F10| F11| F12|BOOT|   |RMOD|
+    +------------------------------------------------------------------------------+
+    |    |    |    |    |    |    |    |    |    |    |    |    |    |        |RM_P|
+    +------------------------------------------------------------------------------+
+    |      |    |    |    |    |    |    |    |    |    |    |    |    |      |    |
+    +-------------------------------------------------------------------      -----+
+    |       |    |    |    |    |    |    |    |    |    |    |    |    |     |    |
+    +------------------------------------------------------------------------------+
+    |     |    |    |    |    |    |    |    |    |    |    |    |       |RSAI|    |
+    +------------------------------------------------------------------------------+
+    |     |     |     |                             |     |    |    |RHUD|RSAD|RHUI|
+    +------------------------------------------------------------------------------+
+*/
+  [MAC_FN] = LAYOUT_75_iso(
+  /*  0           1           2           3           4           5           6           7           8           9           10          11          12          13          14          15         */
+      _______,    KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,      KC_F6,      KC_F7,      KC_F8,      KC_F9,      KC_F10,     KC_F11,     KC_F12,     QK_BOOT,    _______,    RGB_M_P    ,
```

Comment:
small suggestion. From the docs the convention for `QK_BOOT` seems to to be `Fn+Esc`. This changes this which will likely confuse users that didn't bother reading the keymap thoroughly. I would propose to move it back to the the `Esc` key again.